linode/manager
# upcoming: [M3-7983] - Use 'edge'-class plans for edge regions

## Description 📝
Updates the logic in `PlansPanel` to display plans with the new 'edge' class when deploying a Linode in an edge region.

## Target release date 🗓️
ASAP (out of cycle)

## Preview 📷

|Before|After|
|------|------|
|![Screenshot 2024-04-30 at 1 46 04 PM](https://github.com/linode/manager/assets/122488130/8f077623-f763-4316-a681-7a9aa20ea395)|![Screenshot 2024-04-30 at 1 43 36 PM](https://github.com/linode/manager/assets/122488130/102c4a3b-eae1-4a7a-a223-13a2be2bf314)|

* *Note: The "Edge Nanode 1" plan is used in internal testing only.*

## How to test 🧪

### Prerequisites
- Add the `edge_compute` and `edge_testing` customer tags.

### Verification steps
- Enter the Linode Create flow
- Select an edge region
- Verify only 'edge'-class plans are displayed

## As an Author I have considered 🤔

*Check all that apply*



---
